schema/rdl_schema_0.1.json

```diff
@@ -133,6 +128,22 @@
       },
       "minItems": 1,
       "uniqueItems": true
+    },
+    "sources": {
+      "title": "Sources",
+      "type": "array",
+      "description": "Sources of information or predictions that contribute to the creation of the dataset.",
```

Comment:
Why are we using 'predictions' here? I would prefer we don't use this word relating to disaster risk information, it is very loaded and not something we are providing! 'Sources of information that...' would be fine.

Reply:
I think that was an atttempt to make it clear that the source might be simulated future data, i.e. a prediction. Is there a better less loaded word or is it that you think users of this data will be aware enough that "information" might be 'real' or 'simulated'?

Reply:
I thought so, that was my concern! We simulate both current and future data (projections). Even data on real events in this standard have been simulated, so there is not a distinction needed in this description. For future risk we would say projection or in some cases forecasting, but we don't need to specify that here. Rather say 'information or data'?